docs/pages/versions/v46.0.0/sdk/cellular.md

```diff
@@ -16,6 +18,48 @@ import PlatformsSection from '~/components/plugins/PlatformsSection';
 
 <APIInstallSection />
 
+## Configuration in app.json / app.config.js
+
+You can configure `expo-cellular` using its built-in [config plugin](../../../guides/config-plugins.md) if you use config plugins in your project ([EAS Build](../../../build/introduction.md) or `expo run:[android|ios]`).
```

Comment:
```suggestion
To configure `expo-cellular`, use the built-in [config plugin](/guides/config-plugins) in the **app.json** or **app.config.js** for [EAS Build](/build/introduction) or with `npx expo run:[android|ios]`. The plugin allows you to configure properties that cannot be set at runtime and require building a new app binary to take effect.
```

I think we can be a little bit explicit about what config plugins do, which file to specify them and when the condition of creating a new binary might be required.

Also, putting EAS Build and `npx expo run` commands in brackets feels slightly off.